pages/en/contrib/style-guide.md

```diff
@@ -24,6 +25,152 @@ Style conventions for this document:
 
 ## General guidelines
 
+{% include important.html content="
+
+The rules described in this document are evolving, therefore not all of our
+code-base is following them yet. This is expected and OK. Our approach is to
+incrementally improve the parts that are changing most often, i.e. fix the
+coding style only as part of changes needed for a bug fix/new feature.
+
+" %}
+
+### Variable declarations
+
+Prefer to use immutable variables declared with `const` keyword:
+
+### Good
+
+```js
+const user = app.models.User;
+const saltWorkFactor = user.settings.saltWorkFactor;
```

Comment:
Arguably even better: `const { saltWorkFactor } = user.settings;`

Aside: I believe `user` should be `User`?